### ISSUE TITLE
VSCode `Fix All` command doesn't trigger `ESLint: Fix all auto-fixable Problems`

### ISSUE THREAD

**Issue by tjx666** [microsoft/vscode-eslint#1428]
VSCode: 1.65.0-insider
OS: MacOS12.2.2
eslint: pre-rleasev2.2.3

**Comment by tjx666** [comment]
`editor.codeActionsOnSave` is not related with my use case. I use autoSave after delay.

**Comment by dbaeumer** [comment]
Here is what I am seeing. Is this not working for you?

![cast](https://user-images.githubusercontent.com/1931590/175330953-9c14d71d-e89b-4ed2-b6b2-8471c61c1bdb.gif)

**Comment by tjx666** [comment]
My use case is call:

<img width="803" alt="image" src="https://user-images.githubusercontent.com/41773861/175827505-92c708e3-2ede-48f3-a673-7a3032fbe628.png">

**Comment by dbaeumer** [comment]
@mjbvz do you know where the `Fix All` comes from (e.g. TypeScript)?

**Comment by dbaeumer** [comment]
@mjbvz thanks!

@tjx666 the `Fix All` label is TS specific. I will close the issue since from an ESLint perspective things work as expected.

**Comment by tjx666** [comment]
You can see there is a command `Fix All`:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/176235418-02544cae-d262-410c-98ee-d42534bc52b4.png">

I's for editor, not only js/ts:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/176235522-126ecebb-332d-4790-8259-e84c182def40.png">

In follwing situation, there is an eslint error and a stylelint error:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/176235849-ccfe7938-57ef-4d8f-a8a1-852f1d4efb05.png">

When I call `Fix All`, only stylelint auto fix is triggered:

https://user-images.githubusercontent.com/41773861/176236153-6f37e513-29b8-427a-9caf-4046034d2d70.mov

**Comment by tjx666** [comment]
@dbaeumer I'm sorry that my origin issue is not about the `Fix All` label in following:

 ![](https://user-images.githubusercontent.com/41773861/175827505-92c708e3-2ede-48f3-a673-7a3032fbe628.png)

**Comment by dbaeumer** [comment]
I am not sure I fully understand this. When I trigger `Fix All` I get the same submenu poping up. In the latest version it now shows that the Fix All entry is for TS.
![cast](https://user-images.githubusercontent.com/1931590/176436733-7b360f2f-ee00-45f0-9878-d397ab31362e.gif)

**Comment by dbaeumer** [comment]
Hmm, strange since this works for me.

![cast](https://user-images.githubusercontent.com/1931590/176634100-0e235669-fb07-4e67-bf83-300fd46dbbd0.gif)

Could you provide me with a GitHub repository that has your setup. May be it is a setting / configuration issue

**Comment by tjx666** [comment]
@dbaeumer

You need to enable `autoSave` and disable `codeActionsOnSave`:

```jsonc
{
    "files.autoSave": "afterDelay",
    // not work when enable autoSave
    // "editor.codeActionsOnSave": {
    //     "source.fixAll": false,
    //     "source.addMissingImports": true,
    //     "source.fixAll.eslint": true,
    //     "source.fixAll.stylelint": true
    // },
}
```

**Comment by dbaeumer** [comment]
Was able to reproduce this.

**Comment by dbaeumer** [comment]
Fix is in 2.2.5 pre-release
